src/views/exercicios_POST.html

```diff
@@ -4,27 +4,52 @@
 	<title></title>
 </head>
 <body>
+	
+	<form class="form" id="myForm">
+		<label for="inp_nome_exercicio">Nome do Exercício</label>
+		<input type="text" name="nome_exercicio" id="inp_nome_exercicio">
+		<br>
+		<label for="inp_nome_equipamento">Nome do equipamento</label>
+		<input type="text" name="nome_equipamento" id="inp_nome_equipamento">
+		<br>
+		<label for="inp_nome_dificuldade">Nome da dificuldade</label>
+		<input type="text" name="nome_dificuldade" id="inp_nome_dificuldade">
+		<br>
+		<label for="inp_nome_musculo">Nome do Musculo</label>
+		<input type="text" name="nome_musculo" id="inp_nome_musculo">
+		<br>
+		<button type="submit">Inserir</button>
+	</form>
+
 	<script>
-		/*
-		fetch('http://localhost:3000/api/exercicio/', {
-			method: 'POST',
-			headers: {
-				'Content-Type': 'application/json'
-			},
+		const myForm = document.getElementById('myForm');
+
+		myForm.addEventListener('submit', function(e) {
+			e.preventDefault();
+
+			const formData = new FormData(this);
+
+			const formDataObject = Object.fromEntries(formData.entries());
+			const formDataJsonString = JSON.stringify(formDataObject);
```

Comment:
Converte form data para javascript object e converter em JSON 
https://www.learnwithjason.dev/blog/get-form-values-as-json
https://stackoverflow.com/questions/41431322/how-to-convert-formdata-html5-object-to-json